ui/app/src/utils/itemActions.ts

```diff
@@ -970,7 +970,10 @@ export const itemActionDispatcher = ({
   }
 };
 
-export function editControllerActionCreator(systemType: SystemType, contentTypeId: string): StandardAction {
+export function editControllerActionCreator(
+  systemType: SystemType,
+  contentTypeId: string
+): { payload: EditFilePayload; type: string } {
```

Comment:
Why not use StandardAction with its generics?
```suggestion
): StandardAction<EditFilePayload> {
```